### PR TITLE
pelux.xml: bump pelux revisions

### PR DESCRIPTION
**Pull request by TSAnsari**
Point to latest master revisions:
meta-pelux
meta-bistro
meta-template

Point to latest qt 5.12 revisions:
meta-qt5
meta-boot2qt

Signed-off-by: Tariq Ansari <tansari@luxoft.com>

### CODE REVIEW

File: pelux.xml [hunk 1]
```diff
@@ -71,13 +71,13 @@
...
            upstream="5.12"
-           revision="6830447387d2bfaef1e855040068df2a7f19d485"
+           revision="b676fea3938221781bf81a8cfcf40c7d2046f8d8"
            name="yocto/meta-qt5"
            path="sources/meta-qt5"/>
 
   <project remote="code.qt"
            upstream="sumo"
-           revision="880ecb7a6e993a986d24c9a8b813bf2a48f51d24"
+           revision="5e9819044b5536eea9ad01cce0a6cc85d8b1ea6b"
```
Comment (JEderonn) [inline pelux.xml]: I would prefer to bump to 5.12 75e93a6ecb121d9aa0ee4aa986b3eef0dd20ea22 so we have a "stable" version.

File: pelux.xml [hunk 2]
```diff
@@ -71,13 +71,13 @@
   <!-- Qt + Qt Automotive support stuff -->
   <project remote="code.qt"
            upstream="5.12"
-           revision="6830447387d2bfaef1e855040068df2a7f19d485"
+           revision="b676fea3938221781bf81a8cfcf40c7d2046f8d8"
```
Comment (JEderonn) [inline pelux.xml]: I would prefer to not bump this since it's already 5.12.

**Comment by TSAnsari** [comment]
The build failed on Jenkins due to gammaray issue. Previously, there was a fix bbappend file added for gammaray in meta-pelux. However, that issue has now been fixed in qt5.12, so the fix in meta-pelux is failing the build. Removing this fix from meta-pelux in PR 
https://github.com/Pelagicore/meta-pelux/pull/241
Once the gammaray fix is merged in meta-pelux, this bump should work fine.

**Comment by TSAnsari** [comment]
So it turns out that we need to make some changes to the meta-pelux neptune.service file, which overrides the neptune.service file in meta-boot2qt layer. Some config file names have changed in newer versions of qt as well as how neptune3 is started, so we need to update meta-pelux service file to reflect that. So  this PR is on hold until I fix the service file in the meta-pelux layer.

**Pull request merged**